scripts/jenkins/Jenkinsfile.release

```diff
@@ -7,6 +7,9 @@ properties([
         string(name: 'TAG',
                defaultValue: 'v0.0.0',
                description: 'CLUO tag to release'),
+        booleanParam(name: "PUSH_LATEST",
```

Comment:
What's the rationale for this? Can we just decide to always have a "latest" tag that is the most recent published version or just never have one.